docs/UserGuide.md

```diff
@@ -321,19 +321,19 @@ Expected outcome:
 | **List**             | `list`                           |
 | **Clear**            | `clear`                          |
 | **Exit**             | `exit`                           |
-| **Add**              |                                  |
+| **Add**              | `add n/COMPANY_NAME p/POSITION e/EMAIL w/WEBSITE s/SALARY [t/TAG]… `, e.g. `add n/Microsoft p/Software Engineer e/hr@microsoft.com w/https://careers.microsoft.com s/5000 t/Urgent`                                 |
 | **Delete**           |                                  |
 | **Edit**             |                                  |
-| **Status**           | `status`, `status 1 o`           |
-| **Remark**           | `remark`, `remark 1 r/ hello`    |
+| **Status**           | `status`, e.g. `status 1 o`           |
+| **Remark**           | `remark`, e.g. `remark 1 r/ hello`    |
 | **Find by Company**  |                                  |
 | **Find by Position** |                                  |
 | **Filter**           |                                  |
 | **Sort**             | `sort ORDER`<br> e.g.`, `sort a` |
```

Comment:
Is it possible to change this to `sort ORDER`, e.g `sort a` to standardise? Thank you.

Reply:
Thank for the suggestion, it has now been changed.